### DISCUSSION TITLE
[Taskbar notification icon spacing] rename

### DISCUSSION THREAD

**Discussion by mateusfn98** [ramensoftware/windhawk-mods#502] [General]
I think "Taskbar Tray Icon Spacing" is more intuitive.

**Comment by m417z** [comment]
The [official name](https://devblogs.microsoft.com/oldnewthing/20030910-00/?p=42583) of the icons is "notification icons", but yeah, it's messy and perhaps "tray icon" is more popular. I'll wait for more feedback and maybe I'll rename it in the future.

**Reply by stamminator** [reply]
I'm comfortable with using the unofficial name "tray". If a user hears "notification area", they're likely to think of this:

![GFOB2Kh4td](https://github.com/user-attachments/assets/e145ffda-6965-4dff-8062-3b97f8acb779)

They might also think that "notification icon" refers to the image in a notification. Aside from that, the name also has less semantic distinction than before, since regular taskbar icons can display limited notification info via badges, progress, and color.

If I hear someone say "system tray" or "taskbar tray", I know exactly what they're talking about. Same with "tray icon". It may not be Microsoft-approved, but it's more distinct and probably more recognized.

EDIT: For what it's worth, [this Windows Insider blog post](https://blogs.windows.com/windows-insider/2024/06/14/announcing-windows-11-insider-preview-build-22635-3785-beta-channel/) refers to it as the system tray.

**Reply by stamminator** [reply]
Sorry to spam, but I've just found that the Windows 11 settings menu actually calls them system tray icons as well. I think that's as authoritative as it gets.

![DOjPx99rEt](https://github.com/user-attachments/assets/4552e86c-b797-4e9d-9c5d-265bde037705)

**Reply by m417z** [reply]
Thanks, I'll rename it in the next version.
